.gitignore

```diff
@@ -6,3 +6,5 @@ install/update.log
 .settings/
 *.diff
 *~
+.idea/
```

Comment:
While not strictly belonging to this PR, this is a useful addition for PHPStorm users...